superset/db_engine_specs/base.py

```diff
@@ -521,7 +510,7 @@ def create_table_from_excel(  # pylint: disable=too-many-arguments
         Create table from contents of a excel. Note: this method does not create
         metadata for the table.
         """
-        df = cls.excel_to_df(io=filename, **excel_to_df_kwargs,)
+        df = pd.read_excel(io=filename, **excel_to_df_kwargs)
```

Comment:
Removing method `excel_to_df` as it no longer does anything beyond calling `pd.read_excel()`.